tavla/app/(admin)/edit/[id]/components/TileCard/index.tsx

```diff
@@ -203,9 +203,20 @@ function TileCard({
                             {tile.displayName ?? tile.name}
                         </Heading3>
                         <div className="hidden sm:flex flex-row gap-4 h-8">
-                            {transportModes.map((tm) => (
-                                <TransportIcon transportMode={tm} key={tm} />
-                            ))}
+                            {transportModes
+                                .filter(
+                                    (tm) =>
+                                        !(
+                                            tm === 'coach' &&
+                                            transportModes.includes('bus')
+                                        ),
+                                )
+                                .map((tm) => (
```

Comment:
Komponenten her er så svær omg 😵‍💫 Kunne det vært lettere å lese om vi flyttet den ut i en egen variabel kalt `uniqTransportModeIcons` og filtrert der? 🤔 If not, gønn inn

Reply:
lurt! fiksa det